lib/ts/recipe/emailpassword/components/library/label.tsx

```diff
@@ -26,7 +26,7 @@ export default function Label({ value, showIsRequired }: LabelProps): JSX.Elemen
     return (
         <div data-supertokens="label">
             {t(value)}
-            {showIsRequired && " *"}
+            {showIsRequired && value !== "" && " *"}
```

Comment:
what if the label has spaces like `"  "`? Shouldn't we trim it? Or is it already trimmed by the time it comes here?

Reply:
added a new commit for same